README.md

```diff
@@ -5,3 +5,13 @@ This role deploys [Storage node][neofs-node] for [NeoFS][neofs].
 
 [neofs-node]: https://github.com/nspcc-dev/neofs-node
 [neofs]:      https://fs.neo.org
+
+### NeoFS Storage Compatibility Table
+
+| NeoFS Node version | ansible-neofs-storage role version | min NeoFS Node supported version | max NeoFS Node compatible version |
+|--------------------|------------------------------------|----------------------------------|-----------------------------------|
+| 0.30.2             | v0.0.33                            | 0.30.0                           | 0.30.2                            |
```

Comment:
So 0.36.0 node is not compatible with 0.0.33 role? How does it work then?

Reply:
Updated